### PR TITLE
Make _to_multi_line_cmd() public #259

### PR DESCRIPTION
**Pull request by christyanthony**
…ed #TODO in helpers/lib_tasks_utils.py

### CODE REVIEW

**Comment by christyanthony** [comment]
I have reverted the changes and added only script "rename_to_multi_line_cmd.sh" and the change involved when removing TODO in "helpers/lib_tasks_utils.py"

Hope everything works this time:)

**Comment by samarth9008** [comment]
Go through this doc. https://github.com/sorrentum/sorrentum/blob/master/docs/First_review_process.md

**Comment by christyanthony** [comment]
This is the pull request created for the said comments in #336. 
I have ran linter and also ran tests to make sure nothing is broken. Please let me know if there's anything else to do.

**Comment by samarth9008** [comment]
Who will review it?

**Comment by christyanthony** [comment]
> Who will review it?

Apologies. I mis-clicked.

File: helpers/lib_tasks_utils.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def _to_single_line_cmd(cmd: Union[str, List[str]]) -> str:
     return cmd
 
 
-# TODO(Grisha): make it public #755.
+
```
Comment (DanilYachmenev) [inline helpers/lib_tasks_utils.py:157]: remove this empty line
overall the distance between 2 separate functions is 2 lines

**Review by DanilYachmenev — COMMENTED** [review]
1 nit and merge

File: dev_scripts/cleanup_scripts/rename_to_multi_line_cmd.sh
Comment (DanilYachmenev) [inline dev_scripts/cleanup_scripts/rename_to_multi_line_cmd.sh:1]: nah, 1 more nit actually
pls rename according to the name style for scripts in the folder `dev_scripts/cleanup_scripts`
So here it is smth like `SorrTask259_Make__to_multi_line_cmd_public.sh`

**Review by DanilYachmenev — COMMENTED** [review]
lgtm

@samarth9008 when this one is merged could you also check out to a new branch related to this PR and run this script there, please?
The GH action tests will start, we see if they pass and then merge the related changes

**Comment by gpsaggese** [comment]
I'm confused. I see a bunch of refs in the tree that should have been changed.

```
> jackpy to_multi_line_cmd
oms/oms_lib_tasks.py:108:    multiline_docker_cmd = hlitauti._to_multi_line_cmd(cmd)
oms/oms_lib_tasks.py:160:    cmd = hlitauti._to_multi_line_cmd(cmd)
oms/oms_lib_tasks.py:213:    cmd = hlitauti._to_multi_line_cmd(cmd)
optimizer/opt_lib_tasks.py:382:    docker_up_cmd = hlitauti._to_multi_line_cmd(docker_up_cmd_)
optimizer/opt_lib_tasks.py:433:    docker_down_cmd = hlitauti._to_multi_line_cmd(docker_down_cmd_)
im_v2/im_lib_tasks.py:83:    multiline_docker_cmd = hlitauti._to_multi_line_cmd(cmd)
im_v2/im_lib_tasks.py:135:    cmd = hlitauti._to_multi_line_cmd(cmd)
im_v2/im_lib_tasks.py:188:    cmd = hlitauti._to_multi_line_cmd(cmd)
im_v2/im_lib_tasks.py:247:#     multiline_docker_cmd = hlitauti._to_multi_line_cmd(cmd)
im_v2/im_lib_tasks.py:312:#     multiline_docker_cmd = hlitauti._to_multi_line_cmd(cmd)
helpers/lib_tasks_docker.py:1190:        docker_config_cmd_as_str = hlitauti._to_multi_line_cmd(docker_config_cmd)
helpers/lib_tasks_docker.py:1197:    docker_cmd_: str = hlitauti._to_multi_line_cmd(docker_cmd_)
helpers/lib_tasks_docker_release.py:118:    # TODO(gp): Use _to_multi_line_cmd()
helpers/lib_tasks_docker_release.py:349:    # TODO(gp): Use _to_multi_line_cmd()
helpers/lib_tasks_utils.py:158:def _to_multi_line_cmd(docker_cmd_: List[str]) -> str:
helpers/lib_tasks_pytest.py:184:    # TODO(gp): Use _to_multi_line_cmd()
```

**Comment by DanilYachmenev** [comment]
> I'm confused. I see a bunch of refs in the tree that should have been changed.

Since interns work at `sorrentum` there is no way to run regular or GH action tests for them so we cannot actually check if the replacements done by the script are not breaking the build
So following up your [comment](https://github.com/sorrentum/sorrentum/pull/345#issuecomment-1605386426) we've decided to merge text replacements scripts for students after they present their work at PRs and then run the scripts by ourselves at `cmamp`

Same as here basically https://github.com/sorrentum/sorrentum/pull/350#issuecomment-1607780757

**Comment by gpsaggese** [comment]
@DanilYachmenev what's the next step here? Can you take over / finalize in cmamp. As discussed we should avoid renaming PRs in Sorrentum.

**Comment by samarth9008** [comment]
I'll run the script on the cmamp side

**Comment by samarth9008** [comment]
https://github.com/cryptokaizen/cmamp/pull/4721

**Comment by DanilYachmenev** [comment]
> [cryptokaizen/cmamp#4721](https://github.com/cryptokaizen/cmamp/pull/4721)

All done here

**Pull request closed without merging**